{
  "0 debug pnpm:scope": {
    "selected": 1
  },
  "1 info pnpm": {
    "err": {
      "name": "Error",
      "message": "not found: io",
      "code": "ENOENT",
      "stack": "Error: not found: io\n    at getNotFoundError (C:\\Users\\bamio\\AppData\\Roaming\\nvm\\v16.13.2\\node_modules\\pnpm\\dist\\pnpm.cjs:41517:51)\n    at Function.whichSync [as sync] (C:\\Users\\bamio\\AppData\\Roaming\\nvm\\v16.13.2\\node_modules\\pnpm\\dist\\pnpm.cjs:41594:13)\n    at getCommandAbsolutePathSync (C:\\Users\\bamio\\AppData\\Roaming\\nvm\\v16.13.2\\node_modules\\pnpm\\dist\\pnpm.cjs:43159:44)\n    at default_1 (C:\\Users\\bamio\\AppData\\Roaming\\nvm\\v16.13.2\\node_modules\\pnpm\\dist\\pnpm.cjs:43168:32)\n    at C:\\Users\\bamio\\AppData\\Roaming\\nvm\\v16.13.2\\node_modules\\pnpm\\dist\\pnpm.cjs:171981:39"
    }
  },
  "2 error pnpm": {
    "code": "ERR_PNPM_RECURSIVE_EXEC_FIRST_FAIL",
    "prefix": "C:\\Users\\bamio\\git\\try-finger\\backend",
    "err": {
      "name": "pnpm",
      "message": "not found: io",
      "code": "ERR_PNPM_RECURSIVE_EXEC_FIRST_FAIL",
      "stack": "Error: not found: io\n    at getNotFoundError (C:\\Users\\bamio\\AppData\\Roaming\\nvm\\v16.13.2\\node_modules\\pnpm\\dist\\pnpm.cjs:41517:51)\n    at Function.whichSync [as sync] (C:\\Users\\bamio\\AppData\\Roaming\\nvm\\v16.13.2\\node_modules\\pnpm\\dist\\pnpm.cjs:41594:13)\n    at getCommandAbsolutePathSync (C:\\Users\\bamio\\AppData\\Roaming\\nvm\\v16.13.2\\node_modules\\pnpm\\dist\\pnpm.cjs:43159:44)\n    at default_1 (C:\\Users\\bamio\\AppData\\Roaming\\nvm\\v16.13.2\\node_modules\\pnpm\\dist\\pnpm.cjs:43168:32)\n    at C:\\Users\\bamio\\AppData\\Roaming\\nvm\\v16.13.2\\node_modules\\pnpm\\dist\\pnpm.cjs:171981:39"
    }
  }
}